recipes/database.rb

```diff
@@ -16,13 +16,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-bash 'database' do
-  user 'root'
-  cwd "#{node['timesync']['application_path']}/source"
-  code <<-EOH
-    npm run migrations
-    chown #{node['timesync']['user']}:#{node['timesync']['group']} dev.sqlite3
-  EOH
+include_recipe 'postgresql::client'
+
+pg = Chef::EncryptedDataBagItem.load(node['timesync']['databag'],
+                                     'pg')
+
+magic_shell_environment 'PG_CONNECTION_STRING' do
+  value "postgres://#{pg['user']}:#{pg['pass']}@#{pg['host']}:#{pg['port']}/" \
+    "#{pg['database_name']}"
+end
+
+magic_shell_environment 'NODE_ENV' do
+  value 'production'
 end
```

Comment:
This is not the correct way to set environment variables for a service. This will write to /etc/profile.d which means any user can read the pg password.